process/templates/WG_NAME_incubating_stage.md

```diff
@@ -1,21 +1,36 @@
 ## Working Group incubation application
 
-### List of regular contributors
-The WG must have a minimum of 5 contributors from at least 3 different organizations attending regulary.
+### List WG Chair(s) and or Vice Chair
+The WG must have a minimum of 1 Chair
 
   * "name, affiliation, GitHub ID"
 
-### TAC sponsor
-The WG must have a TAC sponsor:
 
+### Working Group (WG) has met all Sandbox requirement
+  * "link to sandbox PR"
+
+### List of regular contributors
+The WG must have a minimum of 5 contributors from at least 3 different organizations attending regularly.
```

Comment:
The current TI G&Gs doc sets this [requirement for incubating TIs](https://github.com/ossf/tac/blob/main/process/TI-Gives%2BGets.md#givesrequirements-1) at 3 contributors from at least 2 different orgs. To me it makes sense to increase this requirement for WGs because of their scale, but I also think that we might need to adjust the G&Gs for different TI types to reflect these differences.

Reply:
In fact, the 5 and 3 requirement is also consistent with the [WG lifecycle doc](https://github.com/ossf/tac/blob/main/process/working-group-lifecycle.md#to-become-incubating), so the TI G&Gs need to be adjusted here.